### ISSUE TITLE
XCPretty support for pod repo push

### ISSUE THREAD

**Issue by gal-orlanczyk** [CocoaPods/CocoaPods#10677]
Why isn't it possible to pipe xcpretty to pod repo push and to pod lib lint yes?

## What did you do?

Run `pod repo push specs 'ABC.podspec' --sources='https://cdn.cocoapods.org/,git@private/source.git' --allow-warnings --skip-import-validation --skip-tests --verbose | xcpretty`

## What did you expect to happen?

xcpretty to be able to consume the data (works fine with pod lib lint)

## What happened instead?

nothing...

**Comment by orta** [comment]
xcpretty is for Xcode output, not CocoaPods

**Comment by gal-orlanczyk** [comment]
Yes, but pod repo push uses xcode build under the hood without allowing to minimize the output.
In comparison pod lib lint allows running xcpretty in a pipe operator and make logs more concise.
